KanbanGuidelines.md

```diff
@@ -0,0 +1,49 @@
+# Kanban Guidelines
+
+## Kanban General
+
+A kanban board helps us *visualising work*, *limiting the amount of work in progress*, and *pulling work through the system*.
+The following guidelines will help us standardize our use with Kanban boards, so that we can switch projects efficiently without having to learn a new Kanban method this particular project employs.
+
+## Board layout
```

Comment:
Maybe we could also have some lanes for the backlog itself, something like 'new, needs refinement, ready for development'. to indicate if the task is clear enough to add to the list of active work items?

Reply:
This is more the domain of the PO/PL, and I'm not the right person to describe how other people should do their own job. We should focus on how we work efficiently, and how roles in the system connect to ours.